tests/e2e/specs/plugin-reset.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * WordPress dependencies
+ */
+import { deactivatePlugin, activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Reset plugin', () => {
+	beforeAll( async() => {
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await activatePlugin( 'google-site-kit' );
```

Comment:
The Site Kit plugin should actually be already active during E2E tests (done in the Docker setup). The only exception where it needs to be (de)activated is in the test for the plugin activation itself.